coalib/io/FileFactory.py

```diff
@@ -0,0 +1,127 @@
+import os
+
+from coala_utils.decorators import generate_eq
+
+
+@generate_eq('name', 'timestamp')
+class FileFactory:
+    """
+    The ``FileFactory`` is a wrapper class whose objects contain the
```

Comment:
What does it wrap actually? Maybe don't say it's a wrapper class, just say it provides different views on a file.